packages/cardano-services/src/StakePool/DbSyncStakePoolProvider/DbSyncStakePoolProvider.ts

```diff
@@ -75,6 +87,14 @@ export class DbSyncStakePoolProvider extends DbSyncProvider(RunnableModule) impl
     this.#epochMonitor = epochMonitor;
     this.#metadataService = metadataService;
     this.#paginationPageSizeLimit = paginationPageSizeLimit;
+    this.#responseConfig = merge({
+      search: {
+        metrics: {
+          apy: true
+        }
+      },
+      ...responseConfig
+    });
```

Comment:
A bit confusing having both `disableStakePoolMetricApy` (`responseConfig.search.metrics.apy`) defaulting to `false` and merging the same [config](https://github.com/input-output-hk/cardano-js-sdk/pull/609/files#diff-cfc78f77666af1469e2e670c258fecabed5a923995563490c52370f7a6b4888cR134) with `responseConfig.search.metrics.apy: true`